### PR TITLE
Create MockTimelineInfo

### PR DESCRIPTION
**Pull request by karupayun**
In this PR we are creating MockTimelineInfo. It's a simpler
implementation than the one in TimeGraph and will be used for making
unit tests for CaptureViewElements. For example it's needed in every
Track since Track ask for a TimelineInfo when they are constructed.

The MockTimelineInfo simply assumes that capture start at timestamp = 0
and provides methods to modify Min/Max visible timestamps (which also
updates the MaxCaptureTimestamp). Also provides conversions between
Tick/Us/World using internal width.

http://b/228610504

### CODE REVIEW

**Comment by antonrohr** [comment]
Typos from the commit message:

It's a simpler implementation tha\*n the one in TimeGraph 

The MockTimelineInfo simply assume\*s that capture start at timestamp \*is equal
to 0 and provides method\*s to modify Min/Max visible timestamps (which
also updates the MaxCaptureTimestamp).

File: src/OrbitGl/MockTimelineInfo.h [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    min_visible_ns_ = min_tick;
+    max_visible_ns_ = max_tick;
+    max_capture_ns = std::max(max_capture_ns, max_tick);
+  }
+
+ private:
+  double width_ = 0.f;
+  uint64_t min_visible_ns_ = 0;
+  uint64_t max_visible_ns_ = 0;
+  const uint64_t min_capture_ns = 0;
```
Comment (antonrohr) [inline src/OrbitGl/MockTimelineInfo.h:58]: I think I would remove `min_capture_ns_`. It is only used in `GetCaptureTimeSpanNs`. It feels a little weird that it is used in that method, but nowhere else (for example: `GetNsSinceStart`)

**Review by antonrohr — APPROVED** [review]

**Review by beckerhe — APPROVED** [review]
What Anton said. Otherwise LGTM

File: src/OrbitGl/MockTimelineInfo.h (hunk 1 above)
Comment (karupayun) [reply src/OrbitGl/MockTimelineInfo.h:58]: I thought about that but I forgot to modified later, thanks!

**Pull request merged**